data/json/items/armor.json

```diff
@@ -163,6 +163,19 @@
     "material_thickness": 1,
     "flags": [ "WATER_FRIENDLY", "OUTER", "SPLINT" ]
   },
+  {
+    "id": "arm_xlsplint",
+    "type": "ARMOR",
+    "name": "arm splint XL",
+    "description": "A tool to help set bones and hold them in place.  It is specifically designed to fit Huge people.",
+    "copy-from": "arm_splint",
+    "weight": 600,
+    "volume": 8,
+    "price": 25000,
+    "encumbrance": 140,
+    "warmth": 5,
```

Comment:
Warmth value was not changed compared to basic splint, so it should not be defined here.